code/modules/vending/_vending.dm

```diff
@@ -1611,6 +1615,32 @@ GLOBAL_LIST_EMPTY(vending_machines_to_restock)
 	credits_contained = max(0, credits_contained - credits_to_remove)
 	SSblackbox.record_feedback("amount", "vending machine looted", holochip.credits)
 
+/obj/machinery/vending/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+	if(ishuman(user) && tilted && !held_item)
+		context[SCREENTIP_CONTEXT_LMB] = "Right the vending machine"
+		return TRUE
```

Comment:
```suggestion
	if(ishuman(user) && tilted && isnull(held_item))
		context[SCREENTIP_CONTEXT_LMB] = "Right the vending machine"
		return CONTEXTUAL_SCREENTIP_SET
```

Reply:
Also I think cyborgs can right vendors too?

Reply:
That seems to be correct
![image](https://github.com/tgstation/tgstation/assets/42909981/930af593-42d8-4d79-9eef-c54499f61463)
![image](https://github.com/tgstation/tgstation/assets/42909981/2ef11797-1a17-42a8-86d1-0a56352f915b)